model/utils/data_feed.py

```diff
@@ -7,38 +7,24 @@
 import pandas as pd
 import data.mock_data # this is necessary to create mock data if not existent
 
-# pylint: disable=too-few-public-methods
+
 class DataFeed:
     """
     Performs data conversion for generators
     """
-
-    # TODO Build data_feed
-    def __init__(self, data_folder, data=None, length=0):
+    def __init__(self, data_folder):
         self.data_folder = data_folder
-        self.data = data
-        self.length = length
+        self.data = None
+        self.length = None
 
-    def load_historical_data(self, file_name):
+    def load_historical_data(self, data_file):
         """
-        Parser to read prices and turn them into log-returns"""
-        # TODO parser
-
-        if file_name[-3:] == "csv":
-            historical_data = pd.read_csv(Path(self.data_folder, file_name))
-        elif file_name[-3:] == "prq":
-            historical_data = pd.read_parquet(Path(self.data_folder, file_name))
+        reads historical log returns
+        """
+        historical_data = pd.read_parquet(Path(self.data_folder, data_file))
         self.data = np.array(historical_data)
         self.length = len(historical_data)
 
 
-# TODO Quick and dirt hack to load historical data only once
-data_folder = Path(__file__, "../../../data/")
-print(data_folder.resolve())
-data_feed = DataFeed(data_folder=data_folder.resolve())
-try:
-    data_feed.load_historical_data('mock_logreturns.prq')
-except FileNotFoundError as e:
-    raise RuntimeError(
-        "Run `python data/mock_data.py` to generate the mock_logreturns.prg file"
-        ) from e
+data_folder = Path(__file__, "../../../data/").resolve()
+data_file = "mock_logreturns.prq"
```

Comment:
Move these two to the top of the file and make them constants
```
DATA_FOLDER = Path(__file__, "../../../data/").resolve()
DATA_FILE = "mock_logreturns.prq"
```

Reply:
Just thinking. With mock_data it does not matter of course. But in general you want the data file to be flexible with both name and format. At least an csv option. Most users will probably want to try it on csv. I understand that it won't work in `system_parameters.py`. But how about `simulation.configuration.py`?